test/Libraries/Microsoft.Extensions.AI.Evaluation.Reporting.Tests/ScenarioRunResultTests.cs

```diff
@@ -38,6 +38,7 @@ public void SerializeScenarioRunResult()
             messages: [new ChatMessage(ChatRole.User, "prompt")],
             modelResponse: new ChatResponse(new ChatMessage(ChatRole.Assistant, "response")),
             evaluationResult: new EvaluationResult(booleanMetric, numericMetric, stringMetric, metricWithNoValue));
+        Assert.Equal(Defaults.ReportingFormatVersion, entry.FormatVersion);
```

Comment:
If it remains nullable, this should probably also validate what behavior you expect when FormatVersion is set to null and then read again. Does it get null? Does it get the default?

Reply:
If FormatVersion is set to null, I'd expect it to remain null. We don't expect users to change the value, but if they did then it would be whatever they set it to.